source/Octopus.Client/Properties/AssemblyInfo.cs

```diff
@@ -3,6 +3,4 @@
 using System.Runtime.CompilerServices;
 
 [assembly: AssemblyTitle("Octopus.Client")]
-[assembly: InternalsVisibleTo("Octopus.Cli")]
```

Comment:
This is what was causing most of the pain - we were exposing all our internals, including the internalised copy of newtonsoft.json